### DISCUSSION TITLE
Stream cannot be played from Edge Node in Ant Media Server Cluster, How we can solve it?

### DISCUSSION THREAD

**Discussion by asharamseervi** [ant-media/Ant-Media-Server#4987] [FAQs] [answered]
Hi,

I have the Ant Media Server cluster up and running at AWS and I can play a stream from Origin Node.
While I'm not able to play the same stream on Edge nodes.

I have checked and validated the security groups and the UDP port range 50000-60000 is open.

**Comment by Mohit-3196** [comment, marked as answer]
Thank you for your question.
It looks like the TCP port 5000 is not open for internal communication.
Also, you need to open this port in only cluster mode for internal network communication so that the edge group of nodes can communicate with the origin group of nodes. It should not be open to the public.
You can refer [to this](https://antmedia.io/docs/guides/installing-on-linux/installing-ams-on-linux/#server-ports) for a complete list of server ports.

Thank you.
